tests/conftest.py

```diff
@@ -60,11 +61,36 @@
 # these tests are run separately
 collect_ignore_glob = ["docs/*.py"]
 
+# Defines how tests are parallelized in the CI
+PATH_PYTEST_MARKER_MAPPINGS = {
```

Comment:
Maybe it would be cleaner to have a conftest in each directory which specifies the marker for that path?

Reply:
This is not only directories but also single files. It's possible to do markers for entire files (maybe also packages) but I decided against it because then
* everything is spread out and it's hard to see at a glance how things are mapped
* might be possible that some (new) test files / directories don't get a marker and then they aren't run

Reply:
I was suggesting just handling the mapping via different conftests and still having the fixture that sets the marker (with a default) here. I.e. in the different conftests you set the constant `PYTEST_MARKER` if it's not already been set. You could also set it at the top of a test file.

_But_ on second thought, it probably is clearer having it all in one place. We just need to be judicious about updating it (e.g. when we get a new directory).